src/core/session.cpp

```diff
@@ -15,9 +15,6 @@
 using namespace soci;
 using namespace soci::details;
 
-namespace // anonymous
```

Comment:
Was this removed intentionally?

Reply:
Yes, as a workaround for a MSVC bug. See https://github.com/SOCI/soci/pull/1118/commits/efe3a055238418ec61c5d61a74bd7f10dc5931c5 for details